pkg/kubelet/server.go

```diff
@@ -65,8 +65,6 @@ func ListenAndServeKubeletServer(host HostInterface, address net.IP, port uint,
 	s := &http.Server{
 		Addr:           net.JoinHostPort(address.String(), strconv.FormatUint(uint64(port), 10)),
 		Handler:        &handler,
-		ReadTimeout:    5 * time.Minute,
-		WriteTimeout:   5 * time.Minute,
```

Comment:
this drops the timeouts on the kubelet server, but I don't see how the new TimeoutHandler is getting set up for the kubelet, only the API server... am I missing something?